### ISSUE TITLE
loss functions

### ISSUE THREAD

**Issue by hallalay** [INDA25PlusPlus/hermanha-ml#5]
MSE, Cross entropy loss?
